tests/python/pants_test/base/test_cmd_line_spec_parser.py

```diff
@@ -68,36 +83,36 @@ def test_descendant(self) -> None:
   def test_files(self) -> None:
     # We assume that specs with an extension are meant to be interpreted as filesystem specs.
     for f in ["a.txt", "a.tmp.cache.txt.bak", "a/b/c.txt", ".a.txt"]:
-      self.assert_filesystem_spec_parsed(f)
-    self.assert_filesystem_spec_parsed("./a.txt", expected_glob="a.txt")
-    self.assert_filesystem_spec_parsed("//./a.txt", expected_glob="a.txt")
+      self.assert_filesystem_spec_parsed(f, literal(f))
+    self.assert_filesystem_spec_parsed("./a.txt", literal("a.txt"))
+    self.assert_filesystem_spec_parsed("//./a.txt", literal("a.txt"))
 
   def test_globs(self) -> None:
-    for glob in ["*", "**/*", "a/b/*", "a/b/test_*.py", "a/b/**/test_*"]:
-      self.assert_filesystem_spec_parsed(glob)
+    for val in ["*", "**/*", "a/b/*", "a/b/test_*.py", "a/b/**/test_*"]:
```

Comment:
I usually call these `glob_string` or something, but I have no issue with this change.

Reply:
Great suggestion! I had no idea what to call this.